src/rest/handler_template.erl

```diff
@@ -30,12 +31,15 @@
 init(Req, State) ->
     {cowboy_rest, Req, State}.
 
-content_types_provided(Req, State) ->
-    {[{<<"application/json">>, to_json}], Req, State}.
+allowed_methods(Req, State) ->
+    {[<<"POST">>], Req, State}.
 
 content_types_accepted(Req, State) ->
     {[{<<"application/json">>, from_json}], Req, State}.
 
+content_types_provided(Req, State) ->
```

Comment:
Tutaj nazywasz funkcję w sposób który nie pasuje do konwencji z której korzystasz w reszcie kodu
```suggestion
provided_content_types(Req, State) ->
```

Reply:
Powtórka: to obowiązkowy cowboy-owy callback